src/ui-router-tabs.js

```diff
@@ -99,9 +102,9 @@ angular.module('ui.router.tabs').directive(
 }]
 ).run(
 ['$templateCache', function($templateCache) {
-    var DEFAULT_TEMPLATE = '<div><uib-tabset class="tab-container" type="{{type}}" vertical="{{vertical}}" ' +
+    var DEFAULT_TEMPLATE = '<div><uib-tabset class="tab-container" active="activeTab" type="{{type}}" vertical="{{vertical}}" ' +
       'justified="{{justified}}">' + '<uib-tab class="tab" ng-repeat="tab in tabs" heading="{{tab.heading}}" ' +
-      'active="tab.active" disable="tab.disable" ng-click="go(tab)">' +
+      'disable="tab.disable" index="$index" ng-click="go(tab)">' +
```

Comment:
why is index required here, won't the forEach above just use the order of the scope.tabs array?
Also $index can be dangerous if you are using filters on the ng-repeat and passing it around.